webpack.config.js

```diff
@@ -8,6 +8,7 @@ const dashLibraryName = packagejson.name.replace(/-/g, '_');
 
 module.exports = {
     entry: { main: './src/lib/index.js' },
+    target: ['web', 'es5'],
```

Comment:
FYI in Dash we've now [gotten rid of this](https://github.com/plotly/dash/commit/c924e420b936f02fdf8a1866d4e10c75ddd4c9bc#diff-1c4a077eebb6590bb428e7a4c12330646293c4532fcaa91c1a561aeef544ae25L4) and instead just target [the last 7 years of supported browsers](https://github.com/plotly/dash/commit/c924e420b936f02fdf8a1866d4e10c75ddd4c9bc#diff-04a108102c9e43f917da603437310001e87eab7885b35930bb55d0e9b584772eR89-R91)